interfaces/automated-testing/rid/ingestion.yaml

```diff
@@ -0,0 +1,163 @@
+            The area of this view: lat1,lng1,lat2,lng2.  The view is the smallest box bounded by
+            the provided corner points.
+          schema:
+            type: string
+          in: query
+          required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetDisplayDataResponse'
+          description: Current remote ID display data was retrieved successfully.
+      operationId: getDisplayData
+      summary: Poll display data
+      description: Request current remote ID display data as would be visualized by a Display Application.
+  /display_data/{id}:
+    get:
+      parameters:
+        -
+          examples:
+            default:
+              value: '1e3adb99-acc9-424f-a04e-a0743538849a'
+          name: id
+          description: >-
+            ID of flight identified in `/display_data`.
+          schema:
+            type: string
+          in: path
+          required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetDetailsResponse'
+          description: Details about the requested flight were retrieved successfully.
+        '404':
+          description: Requested flight was not found.
+      operationId: getDetails
+      summary: Get flight details
+      description: Get the details of a flight identified in `/display_data`.
+components:
+  schemas:
+    GetDisplayDataResponse:
+      description: Response to a request for current data that would be visualized by a Display Application.
+      type: object
+      properties:
+        flights:
+          description: Current information for set of discovered flights whose precise locations are known.
+          type: array
+          items:
+            $ref: '#/components/schemas/Flight'
+          default: []
+        clusters:
+          description: >-
+            Current information for sets of discovered flights whose precise locations are not
+            known.
+          type: array
+          items:
+            $ref: '#/components/schemas/Cluster'
+          default: []
+    GetDetailsResponse:
+      description: Response to a request to get details about a flight.
+      type: object
+    Flight:
+      description: ''
+      required:
+        - id
+        - most_recent_position
+      type: object
+      properties:
+        id:
+          description: >-
+            Identifier of flight that may be used to obtain details about the flight.
+            This is not necessarily the UTM/flight ID in the remote ID system.
+          type: string
+        most_recent_position:
+          description: Most recent position known for the flight.
+          allOf:
+            - $ref: '#/components/schemas/Position'
+        recent_paths:
+          description: Paths the flight recently traveled, if available.
```

Comment:
Suggestion to specify max /  min number of positions in the path, I know this is optional field.

Reply:
This is the ingestion interface, so is providing the data available in the remote ID UTM system to the automated testing framework.  An implementation of this interface is merely relaying information received from other participants in the system to the automated testing framework, so it has no power to ensure that a certain amount of data is populated (because those fields are populated, or not, by other participants).  Specifically, the ingestion interface implementation has no power/ability to ensure that the number of recent positions satisfies a minimum or maximum.  With regard to maximum, and important case we would want to check for is that no more than 60 seconds of recent positions are provided (because that is a requirement in the standard) so we certainly don't want to set a maximum at the ingestion interface as that might cause the ingestion interface implementation to discard an error condition that we want to detect.

I think this raises the question of whether most_recent_position should actually be required at this level, and I think the answer is no -- I will adjust that.

Reply:
A query: Why does it return `most_recent_positions` and not the whole `aircraft state`? Maybe I am getting confused.